### PR TITLE
'make update_docker' should call 'update_deps'

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

File: src/olympia/editors/helpers.py [hunk 1]
```diff
@@ -557,6 +557,7 @@ def get_actions(self, request, addon):
             acl.action_allowed(request, 'ReviewerAdminTools', 'View'))
         reviewable_because_submission_time = (
             not is_limited_reviewer(request) or
+            not addon.latest_version or
```
Comment (diox) [inline src/olympia/editors/helpers.py]: Wrong PR for this commit ?

Comment (wagnerand) [inline src/olympia/editors/helpers.py]: Yeah, saw it right after pushing. Corrected, thanks!

**Comment by EnTeQuAk** [comment]
This is nearly identical to `full_update`, can we just call that in the `update_docker` command? I feel like that makes more sense.

**Comment by wagnerand** [comment]
`full update` does not work for developers that don't have push access to this repo as `update_code` pulls from `origin` only, where as the common case is that developers forked this repo to their namespace and therefore pull from `upstream`.

**Comment by EnTeQuAk** [comment]
indeed, I already updated my comment :)

**Comment by wagnerand** [comment]
Thank you!

**Pull request merged**